DependencyInjection/CmfSeoExtension.php

```diff
@@ -27,6 +26,9 @@
  */
 class CmfSeoExtension extends Extension
 {
+    const CLASS_SEO_METADATA_PHPCR = 'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SeoMetadata';
+    const CLASS_SEO_METADATA_ORM = 'Symfony\Cmf\Bundle\SeoBundle\Document\SeoMetadata';
```

Comment:
And ... the persistence of both (PHPCR and ORM) by the AdminView, we
don't have both. Cause i think the effect will be visible on clicking
the "Save" Button only, right?